merlin/utils.py

```diff
@@ -57,6 +57,27 @@
 DEFAULT_FLUX_VERSION = "0.13"
 
 
+def get_priority(api="rabbit", priority="low"):
+    apis = ["rabbit", "redis"]
+    if api not in apis:
+        LOG.error(f"Unrecognized api '{api}'! Options: {apis}")
+    priorities = ["high", "mid", "low"]
+    if priority not in priorities:
+        LOG.error(f"Unrecognized priority '{priority}'! Options: {priorities}")
+    if priority == "mid":
```

Comment:
This works, but I'd probably handle this as an enum to avoid this kind of conditional sprawl, especially since we're handling a return code, and especially if there's a possibility this gets expanded in the future.

Reply:
Would you have the input parameter be a string or an enum?

Reply:
Fixed it